docs/semgrep-code/semgrep-assistant-code.md

```diff
@@ -54,11 +54,18 @@ Semgrep Assistant supports all the same languages as Semgrep Code. See [Supporte
 
 Semgrep gets API permissions to access code on your pre-selected GitHub or GitLab repositories.
 
-* Semgrep Inc. logs and stores the GPT prompts and responses for the sake of performance evaluation, which include source code snippets.
-* Semgrep Inc. sends relevant lines of code to OpenAI's API, where currently, the "relevant lines of code" means lines that are part of the Semgrep finding, plus 10 lines of context on each side. Semgrep Inc. is likely to expand this, potentially to the entire file, as we learn how to pass more useful context.
-* Semgrep Inc stores and retains these code snippets for up to 6 months. Semgrep Inc. will update you with at least a 30-day notice if we make any changes to the retention policy.
-* Semgrep Inc is a paying customer of OpenAI and has a Data Protection Agreement signed with them (provided upon request by [contacting support](/docs/support). The code snippets we upload are be persisted by OpenAI temporarily, following their data usage policies at [Open AI API data Usage Policies](https://openai.com/policies/api-data-usage-policies).
-* For more details, see the [Semgrep Assistant FAQ](https://get.semgrep.dev/assistant).
+* Semgrep logs and stores the GPT prompts and responses for the sake of performance evaluation, which include source code snippets.
+* Semgrep sends relevant lines of code to OpenAI's API, where currently, the "relevant lines of code" means lines that are part of the Semgrep finding, plus 10 lines of context on each side. Semgrep is likely to expand this, potentially to the entire file, as we learn how to pass more useful context.
+* Semgrep stores and retains these code snippets for up to 6 months. Semgrep will update you with at least a 30-day notice if we make any changes to the retention policy.
+* Semgrep is a paying customer of OpenAI and has a Data Protection Agreement signed with them (provided upon request by [contacting support](/docs/support). The code snippets we upload are be persisted by OpenAI temporarily, following their data usage policies at [Open AI API data Usage Policies](https://openai.com/policies/api-data-usage-policies).
+* * Semgrep takes several steps to protect data that is processed by AI since Assistant requires the sharing of code snippets with a third party. First, Semgrep shares code snippets with OpenAI without identifying the customer or repository name. Second, Semgrep only shares the code necessary to enlist the help of GPT in automating the resolution of each specific alert. Finally, Semgrep only accesses source code repositories on a file-by-file basis; it does not need or request org-level access to your codebase.
```

Comment:
```suggestion
* Semgrep, Inc. takes several steps to protect data that is processed by AI because Assistant requires the sharing of code snippets with a third party. Firstly, Semgrep Assistant shares code snippets with OpenAI without identifying the customer or repository name. Secondly, Semgrep only shares the code necessary to enlist the help of GPT in automating the resolution of each specific alert. Finally, Semgrep Assistant only accesses source code repositories on a file-by-file basis; it does not need or request org-level access to your codebase.
```

This should probably be a list. Long paragraphs are difficult for users to skim.